Creating new apply section (#3526)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply/teacher-training-references.md

---
title: Teacher training references
heading: Your teacher training references
description: |-
  Find out which teacher training references you need to provide when you apply to train to be a teacher and what they'll need to include.
related_content:
    Application tips from a teacher training provider : "/blog/application-tips-from-a-teacher-training-provider"
    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
    7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
navigation: 30.15
navigation_title: Teacher training references

> navigation_description: Find out which teacher training references you need to provide and what they should include.

Reviewer: ```suggestion
navigation_description: Find out who is best to give your teacher training references and what they should include.
```
